Testscripts/Linux/utils.sh

```diff
@@ -2622,21 +2624,21 @@ function install_lagscope () {
 	case "$DISTRO_NAME" in
 		oracle|rhel|centos)
 			install_epel
-			yum -y --nogpgcheck install libaio sysstat git bc make gcc wget
```

Comment:
You'll still need the 'make' because cmake will internally use make.

Reply:
I tested pass, probably it has make by default, anyway I will add it back, thanks for review.